custom/ewsghana/handlers/alerts.py

```diff
@@ -293,8 +293,11 @@ def handle(self):
             text = self.msg.text
 
         if not domain.commtrack_enabled:
-            print "tutaj?"
             return False
+
+        if not verified_contact.location_id:
+            self.respond(NO_SUPPLY_POINT_MESSAGE)
+
```

Comment:
ok, thanks - is there any functionality that should still be accessed even if someone doesn't have a location, or should it always stop processing here?